packages/authentication/src/use-cases/db/get-user-by-username.js

```diff
@@ -0,0 +1,15 @@
+// External dependencies
+const boom = require('@hapi/boom');
+const { models } = require('@siiges-services/core');
+const { Logger } = require('@siiges-services/shared');
+
+async function getUserByUsername(usuario) {
+  const user = await models.Usuario.findOne({ where: { usuario } });
+  if (user) return user.dataValues;
+
+  Logger.error(`[auth/getUserByUsername]: Unable to find \
+username ${usuario} in our database`);
+  throw boom.notFound(`Unable to find username ${usuario} in our database`);
+}
+
+module.exports = getUserByUsername;
```

Comment:
Do you need this use case in this package?
If you want to use the models, That should be in a file named usuario in the domain.
Check this part in the user package if you have doubts, and if you need some help, tell me

Reply:
I don't think so, since I'm using a library (sequellize) and one main characteristics of domains are they're agnostic to this things. So I think is more appropriate categorize this like an use case